modules/deployment/src/main/java/org/eclipse/xpanse/modules/deployment/deployers/terraform/config/TerraformServerConfig.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ *
+ */
+
+package org.eclipse.xpanse.modules.deployment.deployers.terraform.config;
+
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+import lombok.Getter;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+
+/**
+ * Configuration class of terraform-boot server.
+ */
+@Configuration
+@Profile("terraform-boot")
+public class TerraformServerConfig {
```

Comment:
I believe we should separate Terraform-Boot from Terraform-Local and avoid mixing them together.